### PR TITLE
New version: qBittorrent.qBittorrent version 4.6.1

### PR DESCRIPTION
**Pull request by crashglass**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127598)

### CODE REVIEW

File: manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+UpgradeBehavior: install
+Protocols:
+- magnet
+FileExtensions:
+- torrent
+ProductCode: qBittorrent
+Installers:
+- Architecture: x86
+  InstallerUrl: https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-4.6.1/qbittorrent_4.6.1_lt20_qt6_x64_setup.exe/download
+  InstallerSha256: 3376A5A23E6C73748050124BDAAC70D861034FEC31B5F7F21E32BDB002727554
```
Comment (jahvari) [inline manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.installer.yaml:17]:
```suggestion
- Architecture: x64
  InstallerUrl: https://sourceforge.net/projects/qbittorrent/files/latest/download
  InstallerSha256: F3C6608DBF3187E6244B3023510A4425DE8AC5D49A9628EE79453CC28EB19CC8

```

File: manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  - WEBUI: Preserve the network interfaces when connection is down (Fabricio Silva)
+  - WEBUI: Add "Add Tags" field for RSS rules (Matic Babnik)
+  - WEBUI: Fix missing error icon (Trim21)
+  - RSS: Add "Rename rule" button to RSS Downloader (BallsOfSpaghetti)
+  - RSS: Allow to edit RSS feed URL (glassez)
+  - RSS: Allow to assign priority to RSS download rule (glassez)
+  - SEARCH: Use python isolate mode (Chocobo1)
+  - SEARCH: Bump python version minimum requirement to 3.7.0 (Chocobo1)
+  - OTHER: Enable DBUS cmake option on FreeBSD (yuri@FreeBSD)
+  - OTHER: Numerous code improvements and refactorings (glassez, Chocobo1)
```
Comment (jahvari) [inline manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.locale.en-US.yaml:97]:
```suggestion
  v4.6.1 changelog:

    FEATURE: Add option to enable previous Add new torrent dialog behavior (glassez)
    BUGFIX: Prevent crash due to race condition when adding magnet link (glassez)
    BUGFIX: Fix Enter key behavior when add new torrent (glassez)
    BUGFIX: Add missing main window icon (iomezk)
    BUGFIX: Update size of selected files when selection is changed (glassez)
    BUGFIX: Correctly handle changing save path of torrent w/o metadata (glassez)
    BUGFIX: Use appropriate icon for "moving" torrents in transfer list (xavier2k6)
    WEBUI: Drop WebUI default credentials (glassez)
    WEBUI: Add I2P settings to WebUI (thalieht)
    WEBUI: Fix duplicate scrollbar on Transfer List (AgentConDier)
    WEBUI: Fix .torrent file upload on iPadOS (Vitaly Cheptsov)
    WEBUI: Fix incorrect subcategory sorting (Bartu Özen)
    WEBUI: Correctly set save path in RSS rules (glassez)
    WEBUI: Allow to request torrents count via WebAPI (glassez)
    WEBUI: Improve performance of getting torrent numbers via WebAPI (Chocobo1)
    WEBUI: Improve free disk space checking for WebAPI (glassez)
    WINDOWS: NSIS: Fixed typo in the installer's hungarian translation (MartinKing01)
    LINUX: Fix invisible tray icon with Qt5 in Linux (thalieht)
    MACOS: Remove "Physical memory (RAM) usage limit" option (Chocobo1)
```

**Review by jahvari — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20231121191344

File: manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.installer.yaml:17]: `x86` validations are performed on 32-bit OSes.

Comment (stephengillie) [reply manifests/q/qBittorrent/qBittorrent/4.6.1/qBittorrent.qBittorrent.installer.yaml:17]: `x86` validations are performed on 32-bit OSes.

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/127924

**Comment by R-Adrian** [comment]
Nov 27th 2023 - qBittorrent v4.6.2 release

version 4.6.2 was released just a week after 4.6.1- seems v4.6.1 has a nasty memory leak bug in WebUI (and other bugs were fixed too)

**Pull request closed without merging**